src/icons/AllSeeingSealEye.tsx

```diff
@@ -0,0 +1,67 @@
+import classnames, { stroke } from 'classnames/tailwind'
+
+const strokeSecondaryDimmed = classnames(stroke('stroke-secondary-dimmed'))
```

Comment:
You don't need classnames here

Reply:
But if you specify `strokeWidth('stroke-1.5')`, then you'll need it, take a look below